src/lib/util/cursor.c

```diff
@@ -402,6 +402,21 @@ void fr_cursor_merge(fr_cursor_t *cursor, fr_cursor_t *to_append)
 	}
 }
 
+/** Return the next item from a cursor that also satisfies an evaluation function.
+ *
+ * @param[in] cursor	to operate on
+ * @param[in] eval	evaluation function
+ * @return the next item satisfying eval, or NULL if no such item exists
+ */
+void *fr_cursor_filter_current(fr_cursor_t *cursor, bool (*eval)(void *))
```

Comment:
To be consistent with the rest of the cursor code there should be two functions `fr_cursor_filter_head` which finds the first instance of an item in the list that matches the filter, and `fr_cursor_filter_next` which finds the next instance of an item in the list that matches the filter.

Reply:
I'll make it consistent.